test/e2e/tests/send-eth.spec.js

```diff
@@ -418,7 +422,10 @@ describe('Send ETH from inside MetaMask to a Multisig Address', function () {
     await withFixtures(
       {
         fixtures: new FixtureBuilder().build(),
-        ganacheOptions,
+        ganacheOptions: {
```

Comment:
The reason this test is flaky is because this test fails sometimes to get gas and the Next button will not be enabled. This behaviour is not noticed on london hardfork (based on other tests and local testing) and since the test `finds the transaction in the transactions list` it doesn't really matter what fork we use.